lib/adapters/autocomplete-adapter.ts

```diff
@@ -267,11 +267,11 @@ export default class AutocompleteAdapter {
   // * `editor` An Atom {TextEditor} used to obtain the necessary text replacement.
   // * `suggestion` An {atom$AutocompleteSuggestion} to set the replacementPrefix and text properties of.
   public static applyTextEditToSuggestion(
-    textEdit: TextEdit | null,
+    textEdit: TextEdit | undefined,
```

Comment:
This TextEdit comes from `CompletionItem.textEdit` which is typed as `TextEdit | undefined`, makes more sense to use `undefined` here